src/exe/cimbar_send/send.cpp

```diff
@@ -77,13 +77,13 @@ int main(int argc, char** argv)
 
 	int window_size_x = cimbar::Config::image_size_x() + 32;
 	int window_size_y = cimbar::Config::image_size_y() + 32;
-	if (!initialize_GL(window_size_x, window_size_y))
+	if (cimbare_init_window(window_size_x, window_size_y) < 0)
 	{
 		std::cerr << "failed to create window :(" << std::endl;
 		return 70;
 	}
-	auto_scale_window();
-	configure(colorBits, ecc, compressionLevel, legacy_mode);
+	cimbare_auto_scale_window();
+	cimbare_configure(colorBits, ecc, compressionLevel, legacy_mode);
```

Comment:
I'm not sure what to do about the split `*_configure()` calls. It's tempting to smash them together, and that's probably the end state. But I need to think about it more.